src/__integration_tests__/postOrder.ts

```diff
@@ -35,15 +35,29 @@ const sdk = new OpenSeaSDK(
 );
 
 suite("SDK: order posting", () => {
+  test("Post Buy Order", async () => {
+    const postBuyOrder = {
+      accountAddress: WALLET_ADDRESS ? WALLET_ADDRESS : "",
+      startAmount: "0.004",
+      asset: {
+        tokenAddress: "0x1a92f7381b9f03921564a437210bb9396471050c",
```

Comment:
i grappled with this when i implemented - should we make the tester pass these in?

Reply:
maybe have them pass it in optionally and default if they dont

Reply:
We probably should pass in the start amount because the tester will need to have that much WETH but the token address shouldn't matter. I just chose the rarest cool-cat

Reply:
yea fair enough